bitmark-references/bitmark-cli/bitmark-cli.md

```diff
@@ -0,0 +1,703 @@
+
+`--before-block|-b NUMBER`       This is used to provide a "time" limit on
+                                 the operation and the transaction
+                                 must be submitted and confirmed
+                                 before this block is present on the
+                                 blockchain.
+-------------------------------  ---------------------------------
+
+### EXAMPLE
+
+~~~
+bitmark-cli --identity=fred --network=testing grant --receiver=barney --share-id=8981eb58c965e2360b3ffeedf47d8f770a2fff7f67e5c348302d839afbf83dee --quantity=1000 --before-block=234765
+~~~
+
+
+## balance
+
+Display the share balances for a particular owner.  The display starts
+at the particular id and outputs `count` items.  To display more take
+the last share id and use in another `balance` command.  (To avoid a
+repeat of that record use the incremented id as the command will
+search for the next used id)
+
+### OPTIONS
+
+-------------------------------  ---------------------------------
+`--owner|-o IDENTITY|ACCOUNT`    To list records for a different account
+                                 either by an identity or a Base58
+                                 account string.  The default is to
+                                 use the global identity.
+
+`--share-id|-s SHARE-ID`         The transaction id of the share record, which
+                                 is used to identify the fractional
+                                 item to show the balance.
+
+`--count|-c NUMBER`              Limit the number of balance records displayed.
+                                 This defaults to 20 records if
+                                 omitted and can be set from 1 to 100.
+-------------------------------  ---------------------------------
+
+### EXAMPLE
+
+~~~
+bitmark-cli --identity=fred --network=testing balance --owner=barney --share-id=8981eb58c965e2360b3ffeedf47d8f770a2fff7f67e5c348302d839afbf83dee --count=10
+~~~
+
+
+## status
+
+Display the status of a particular transaction id to set whether it is:
+
+* Not found = not present on the blockchain or in that node's memory.
+* Pending = awaiting payment confirmation.
+* Verified = payment is confirmed and just waiting to be incorporated into a block.
+* Confirmed = store on the blockchain.
+
+### OPTIONS
+
+-------------------------------  ---------------------------------
+`--txid|-t HEX-ID`               The transaction id from a previous
+                                 issue or transfer.
+-------------------------------  ---------------------------------
+
+### EXAMPLE
+
+~~~
+bitmark-cli --network=testing status --txid=8981eb58c965e2360b3ffeedf47d8f770a2fff7f67e5c348302d839afbf83dee
+~~~
+
+
+## list
+
+Display all the configuration file identities' public data:
+
+* `SK` flag to indicate the encrypted seed is present
+* The short identity name
+* Base58 account string (the encoded public key)
+* Descriptive text
+
+### OPTIONS
+
```

Comment:
If there are no command options, we should say so, so that people don't think it was left out. (The other option is to omit the section.)